autofaiss/external/optimize.py

```diff
@@ -487,6 +488,7 @@ def optimize_and_measure_indices(
     fs, _ = fsspec.core.url_to_fs(indices_folder)
     indices_file_paths = fs.ls(indices_folder, detail=False)
     index_path2_metric_infos: Dict[str, Dict] = {}
+    suffix_width = int(math.log(len(indices_file_paths))) + 2
```

Comment:
Should be log10 and +1

Reply:
Indeed, I mistaken the usage of zfill, will fix.